shared/src/business/useCases/trialSessions/createTrialSessionInteractor.test.ts

```diff
@@ -107,6 +107,24 @@ describe('createTrialSessionInteractor', () => {
     expect(result.isCalendared).toEqual(true);
   });
 
+  it('shoud associate swing trial sessions when needed', async () => {
```

Comment:
Nit: typo on should & could this be a little more descriptive - like `should associate swing trial sessions when the trial session has a swing session` or something?

Reply:
Good idea! TY